content/general/events.textile

```diff
@@ -0,0 +1,390 @@
+---
+title: Reactor Events
+section: general
+index: 20
+languages:
+  - none
+jump_to:
+  Help with:
+    - Available integrations#integrations
+    - Configuring a Webhook#configure
+    - Sources#sources
+    - Single vs Batched requests#batching
+    - Envelopes#envelope
+    - Payload encoding#encoding
+    - Webhook Security#security
+    - Examples#examples
+---
+
+Reactor Events let you easily allow endpoints and services to be notified via an HTTP request following "messages being published":/realtime/messages or "presence events":/realtime/presence (such as members entering or leaving) on "channels":/realtime/channels.
+
+p(tip). Reactor Events are "rate limited":#transport and are suitable for low to medium volumes of updates. If you expect a high volume of events and messages (averaging more than 25 per second), then you should consider using our "message queues":/general/queues or "firehose":/general/firehose as they are more suitable for higher volumes.
+
+Generally, realtime interactions such as subscribing to events and messages are best done using our "realtime client libraries":/realtime, or our "message queues":/general/queues or "firehose":/general/firehose. However, it can sometimes be preferable to be notified over HTTP when events occur such as a user publishing a message or members entering the presence set on a channel. For example, you may want to start pushing content to a channel only when the channel becomes active and there is someone there to receive the data, or they may want to send a welcome message to someone when they first enter a chat channel. Reactor Events are useful for achieving this in both these scenarios.
+
+In addition, various existing systems, such as Azure Functions, Google Functions, and AWS Lambda rely on HTTP events. Reactor Events will allow for simple integration with said systems.
+
+<a href="/images/diagrams/reactor-webhooks-intro.png" target="_blank">
+  <img src="/images/diagrams/reactor-webhooks-intro.png" style="width: 100%" alt="Ably Reactor Events diagram">
+</a>
+
+You can "configure events":https://support.ably.io/support/solutions/articles/3000074406 from the "Reactor tab in your app":https://support.ably.io/solution/articles/3000074406 on a per app basis which can apply to one or more channels in that app. Reactor Events can be filtered by channel naming using a regular expression, for example @^click_.*_mouse$@. This would match the string @click_@ followed by a string followed by @_mouse@, for example, @click_left_mouse@.
+
+h3(#integrations). Available integrations
+
+At present, we have explicit integration with the following services, however you can also specify *any endpoint* for the Reactor Event Rule to use:
```

Comment:
Perhaps instead:

At present, in addition to support for any custom HTTP endpoint, we have ready-made integrations with the following services: